src/libraries/pkg/runtime.native.System.DirectoryServices/netcoreapp.rids.props

```diff
@@ -0,0 +1,12 @@
+﻿<Project>
+  <ItemGroup Condition="'$(GeneratePackage)' == 'true'">
```

Comment:
this file was copied from System.IO.Ports.Native but probably should be updated for windows.